pkg/util/overridemanager/overridemanager.go

```diff
@@ -255,7 +255,10 @@ func applyJSONPatch(obj *unstructured.Unstructured, overrides []overrideOption)
 		return err
 	}
 
-	patchedObjectJSONBytes, err := patch.Apply(objectJSONBytes)
+	applyOption := jsonpatch.NewApplyOptions()
+	applyOption.EnsurePathExistsOnAdd = true
+	applyOption.AllowMissingPathOnRemove = true
```

Comment:
If an `op` is not written correctly now, it will not take effect and the resources will not propagate to the member cluster. This can remind users to configure the `op` correctly.

With your patch, regardless of whether the `op` is correct or not, resources will be propagated to the member cluster as expected. Users may find it difficult to detect if the `op` has taken effect or not. Even if they do notice that the `op` has not taken effect, it is challenging to distinguish whether there was an issue with writing the `op` correctly or if Karmada itself encountered a problem.


/cc @RainbowMango What do you think?

Reply:
I think this another topic how to show the override result to user. It can log out put result. This problem was not introduced by my PR, it alway exist in previous version.

Reply:
The `karmadactl` need a test override capability, like kubectl dry run

Reply:
Apparently, this is a breaking update.should not be easily changed.